templates/controllers/modals/submissionMetadata/issueEntryTabs.tpl

```diff
@@ -23,6 +23,21 @@
 	{rdelim});
 </script>
 {if not $hideHelp}<p class="pkp_help">{translate key="issue.manage.entryDescription"}</p>{/if}
+<form id='submissionSettingsRevisionForm' class='pkp_form'>
+	{if $submissionSettingsRevisions|@count > 1}
+		<div id='editPreviousRevision'>
```

Comment:
Actually, you don't need divs around fbvForm elements -- it should be styled per default and thus always consistent.